### PR TITLE
Print error message when stopping tiller error

### PR DESCRIPTION
**Pull request by orangedeng**
Related issue:
https://github.com/rancher/rancher/issues/19831

### CODE REVIEW

File: pkg/controllers/user/helm/common.go [hunk 1]
```diff
@@ -58,7 +59,12 @@ func helmInstall(templateDir, kubeconfigPath string, app *v3.App) error {
 	defer cancel()
 	addr := common.GenerateRandomPort()
 	probeAddr := common.GenerateRandomPort()
-	go common.StartTiller(cont, addr, probeAddr, app.Spec.TargetNamespace, kubeconfigPath)
+	go func() {
+		err := common.StartTiller(cont, addr, probeAddr, app.Spec.TargetNamespace, kubeconfigPath)
+		if err != nil {
+			logrus.Errorf("got error [%s] while stopping tiller", err.Error())
```
Comment (gitlawr) [inline pkg/controllers/user/helm/common.go]: Put the error message at the end instead of inside the brackets, as it can be long.

Comment (orangedeng) [reply pkg/controllers/user/helm/common.go]: Updated

**Comment by gitlawr** [comment]
LGTM

**Comment by cjellick** [comment]
This doesnt resolve the issue, does it? You just want to print the errors? What is your plan forward to resolve the issue?

**Comment by cjellick** [comment]
i'll merge, but dont know if i should move the issue over. maybe open a separate isue solely for logging?

**Pull request merged**